content/tutorials/token-authentication.textile

```diff
@@ -274,18 +362,47 @@ In this step we'll show you how to issue subscribe only tokens limited to a sing
 
 Let's get started. First we will add two panels, one that contains a simple login form, and the other with log out link to the HTML page. Add the following beneath the @</h1>@ tag in your @index.html@ file:
 
-```[html]
-  <div id="panel-anonymous">
-    <h3>Login</h3>
-    <form action="/login">
-      <input type="text" name="username" placeholder="Enter your username">
-      <input type="submit" value="Login">
-    </form>
-  </div>
-  <div id="panel-logged-in">
-    Your are logged in. <a href="/logout">Log out</a>
-  </div>
-```
+blang[nodejs].
```

Comment:
btw. you can use the syntax `blang[nodejs,ruby]` to share a language block with two languages.  As the Ruby & Node.js ones are identical, that would be preferred